Add switch for serving content via PHP 7.x

lavagetto opened the pull request:
This gets translated to a cookie only sent to the backend servers on the
debug proxy, meaning we're not fiddling with user's cookies

Bug: T176370

hashar commented:
Poke @Paladox

paladox commented:
+1, thanks.

bd808 commented:
Doh! I made these changes myself before I got around to checking my email for the day. See 55cf22654ccca32adf0347d87c6946613e8f6b28. A new v0.8.3 package has been submitted to Mozilla for review.

addshore commented:
@bd808 it looks like I missed one occourance of mw1017 in my PR anyway!
Thanks for your change!

lavagetto committed:
Add switch for serving content via PHP 7.x

This gets translated to a cookie only sent to the backend servers on the
debug proxy, meaning we're not fiddling with user's cookies

Bug: T176370
